pkg/machineproviders/providers/openshift/machine/v1beta1/provider_test.go

```diff
@@ -1881,6 +1881,10 @@ var _ = Describe("MachineProvider", func() {
 					Consistently(komega.Get(machine)).Should(Succeed())
 				})
 
+				It("should not receive an event", func() {
+					Expect(recorder.Events).Should(Not(Receive(ContainSubstring("Deleted"))))
```

Comment:
We expect no event at all right? So you can just say it shouldn't receive 
```suggestion
					Expect(recorder.Events).Should(Not(Receive()))
```

Reply:
Addressed